docs/docs/filename.md

```diff
@@ -428,7 +428,7 @@ If, for some obscure reason, it has both:
 image - image contains the tag solo group is one of the image tags %md5%.%ext%
 ```
 
-If, it does not have solo nor group:
+If, it does not have solo or group:
```

Comment:
Question for my own curiosity: I'm not a native speaker, is "nor" not valid here? I know it's not mandatory since "neither" was not used, but aren't both interchangeable in this context? Or does "or" here sound better?

Reply:
Both "nor" and "or" are valid in this context. However, using "or" here simplifies the sentence and makes it more straightforward. It's a matter of style, and your choice works well.